examples/behaviors/basic/actions/replace/once/index.xml.njk

```diff
@@ -0,0 +1,6 @@
+{% from 'macros/button/index.xml.njk' import button %}
+{% from 'macros/description/index.xml.njk' import description %}
+
+{{ description('Tapping the button below will request an XML fragment and replace the element with the response.') }}
+{# behavior attributes can be attached to a view too #}
+{{ button('Press me', attributes={"action": "replace", "href":"/behaviors/basic/actions/replace/once/_replace_fragment.xml"}) }}
```

Comment:
same here. Applies to every other behavior examples where the raw XML behavior markup is considered to be the "content" and should be surfaced in the file directly, as opposed to the button itself that is the part of the "UI" and ok to abstract out.